src/ZfcDatagrid/Datagrid.php

```diff
@@ -749,11 +760,7 @@ public function getRowStyles()
      */
     public function hasRowStyles()
     {
-        if (count($this->rowStyles) > 0) {
-            return true;
-        }
-
-        return false;
+        return (bool) $this->rowStyles;
```

Comment:
it is the fastest way to check if its empty or not, !empty($foo), is also possible but a bit slower and count($foo) is the slowest way in this case. So for best performance i changed it to cast in bool.